### PR TITLE
Update docs for new console onboarding flow

### PR DESCRIPTION
**Pull request by mattmanning**
This PR is an attempt to bring Rack installation recommendations in line with the new AWS integration and Rack installation procedure in Console.

Effort has also been made to drastically simplify the documentation and recommend as strongly as possible that installations be done through Console. This should result in more Racks being installed in Console where they are more easily managed and avoiding the tedious process of having to add CLI-installed Racks.

## Release Playbook
- [ ] Rebase against master
- [ ] Code review
- [ ] Merge into master
- [ ] Verify changes at http://site-staging.convox.com
- [ ] Promote release on `site-production` in the `convox/production` Rack

### CODE REVIEW

**Comment by mattmanning** [comment]
[Onboarding Checklist Docs Pass](https://trello.com/c/vvlLKzdR/1760-onboarding-checklist-docs-pass)

**Comment by nzoschke** [comment]
Here are notes from a end-to-end QA pass on the docs. Not all are blocking this PR.

## Introduction ideas

- [ ] Make the first docs page "Getting Started".
- [ ] In "Overview" call out philosophy highlights in one paragraph  and link to a philosophy page for full details
- [ ] Import "Getting Started" content as "Quick Start" section
- [ ] Simplify "Getting Started" as much as possible for the happy path.
- [ ] Write as if the UI is very intuitive and has well-designed flow flow. Instead of a paragraph about live-updating, say "Click Install Convox. Configuring AWS will take about 10 minutes. During this you can set up the CLI.".  
- [ ] The links at the bottom for advanced topics are great. Add a link to advanced networking options.
- [ ] Drive product improvements around `convox apps create && watch convox apps info && convox deploy && watch convox apps info` to get first deploy

For example how great would it be if the doc was:

```bash
$ cd rails
$ convox deploy
Creating new app "rails"... OK

Building rails
Creating tarball... OK
Uploading: 736.17 KB / 935.53 KB [=================>----]  78.69 % 1s 
Authenticating 132866487567.dkr.ecr.us-east-1.amazonaws.com: Login Succeeded
running: docker build
Step 1 : FROM convox/rails
Step 2 : COPY Gemfile /app/Gemfile
Step 3 : COPY Gemfile.lock /app/Gemfile.lock
Step 4 : RUN bundle install
...

Deploying rails.
Creating docker registry service... OK
Creating log service... OK
Creating load balancer service... OK
Creating containers... OK
Waiting for health checks... OK
DONE.
```

## Example App

- [ ] convox/rails is not booting on my computer with a Docker reset

```bash
build    │ running: docker tag convox/postgres:latest rails/database
database │ running: docker run -i --rm --name rails-database -p 5432/tcp -v /Users/noah/.convox/volumes/rails/database/var/lib/postgresql/data:/var/lib/postgresql/data rails/database
web      │ running: docker run -i --rm --name rails-web --add-host database:172.17.0.9 -e DATABASE_SCHEME=postgres -e DATABASE_HOST=172.17.0.9 -e DATABASE_PORT=5432 -e DATABASE_PATH=/app -e DATABASE_USERNAME=postgres -e DATABASE_PASSWORD=password -e DATABASE_URL=postgres://postgres:password@172.17.0.9:5432/app -p 0:4000/tcp -p 0:4001/tcp --cpu-shares 10 --memory 536870912 rails/web
database │ FATAL:  database files are incompatible with server
database │ DETAIL:  The data directory was initialized by PostgreSQL version 9.4, which is not compatible with this version 9.5.4.
```

File: _deployment/installing-a-rack.md [hunk 1]
```diff
@@ -29,39 +24,22 @@ Run `convox install --help` to see the available command line options.
...
 ## Installing into an existing VPC
 
 See our [VPC doc](/docs/vpc-configurations#installing-into-an-existing-vpc) for more information.
 
-## What's the difference between installing via Console vs using the CLI?
-
-The CLI installer has some advanced options that aren't available in the web UI yet.
+## Console vs CLI installation
 
 When you install via the CLI, your CLI gets logged into the Rack at the end of a successful installation. That means all commands go straight to the Rack instead of being proxied through Console. After installing a Rack via CLI, you'll need to:
```
Comment (nzoschke) [inline _deployment/installing-a-rack.md]: This is all product warts too. I hope we can eliminate this part of the doc someday.

File: _development/installation.md [hunk 2]
```diff
@@ -24,7 +24,7 @@ Alternatively, on OSX you can also install via Homebrew:
 
     $ brew install convox
```
Comment (nzoschke) [inline _development/installation.md]: Can we document the full procedure and drop the note?

```
$ brew install convox
$ convox update
Updating convox: OK, 20170203184626
```

File: _introduction/getting-started.md [hunk 3]
```diff
@@ -6,40 +6,39 @@ order: 400
...
 
 See [AWS Integration](/docs/aws-integration) for more details.
 
 ## Launch your private PaaS
 
-Next, click on the **Add a Rack** button, followed by **Install a New Rack** in the top navigation bar. Enter a descriptive Rack name such as `production` if you plan to deploy production services, or `development` if this is for testing.
+Next, click on the **Add Rack** button, followed by **Install New** in the top navigation bar. Enter a descriptive Rack name such as `production` if you plan to deploy production services, or `development` if this is for testing. Choose an AWS region in which you'd like to install the Rack from the list of supported regions. It's best to choose a region geographically close to your users.
 
-See [Installing a Rack](/docs/installing-a-rack) for more details.
+A live-updating window will show the installation progress of your Rack in real time, including all of the AWS resources that are created for it. When the installation is done you can dismiss this window, and you will see your new Rack listed with the status "running".
```
Comment (nzoschke) [inline _introduction/getting-started.md]: See above note. I hope we can simplify this because the modal is intuitive enough.

**Comment by mattmanning** [comment]
@nzoschke Thanks for the feedback. Moved the introduction ideas to a new issue at https://github.com/convox/site/issues/496 so it can be broken down into future changes.

**Pull request merged**